tools/rna_tools/pipmir/tool-data/twobit_update.loc.sample

```diff
@@ -0,0 +1,23 @@
+#This is a sample file distributed with Galaxy that is used by some
```

Comment:
twobit.loc is from 2007, two column format, r u sure it is still valid format. I tested on our sytem, it did not work